### PR TITLE
[Tables] Merge samples to one code snippet

### PR DESCRIPTION
**Pull request by YalinLi0312**
- Resolves https://github.com/Azure/azure-sdk-for-python/issues/26901

### CODE REVIEW

File: sdk/tables/azure-data-tables/samples/sample_query_table.py [hunk 1]
```diff
@@ -83,77 +83,42 @@ def sample_query_entities(self):
...
                 queried_entities = table_client.query_entities(
                     query_filter=name_filter, select=["Value"], parameters=parameters
                 )
-
                 for entity_chosen in queried_entities:
                     print(entity_chosen)
-
             except HttpResponseError as e:
-                print(e.message)
+                pass
```
Comment (annatisch) [inline sdk/tables/azure-data-tables/samples/sample_query_table.py:117]: Using `pass` on an HttpResponseError is probably not a good idea for a sample - we don't want people to copy and paste this by accident. I would probably print some kind of message then raise it.

Comment (YalinLi0312) [reply sdk/tables/azure-data-tables/samples/sample_query_table.py:117]: The exception raised here will be caught and print out in main(line131).

Comment (annatisch) [reply sdk/tables/azure-data-tables/samples/sample_query_table.py:117]: No it wont - because this `pass` is suppressing it. It would need to be `raise`

Comment (YalinLi0312) [reply sdk/tables/azure-data-tables/samples/sample_query_table.py]: `raise` is what I meant, updated!

**Review by annatisch — APPROVED** [review]

**Pull request merged**